### ISSUE TITLE
PSNR of pre-trained model is low

### ISSUE THREAD

**Issue by emc222** [OPEN-AIR-SUN/mars#81]
I test the KITTI-MOT dataset pre-trained model as below, but only get 24.1 psnr on 1/4 test set or full 512 frame test set, what is the reason?

![image](https://github.com/OPEN-AIR-SUN/mars/assets/41288487/2fcc6c83-8575-47ad-ac66-aebf72b6c8f9)

### my command
ns-eval --load-config outputs/NSGKITTI-0006-05-260cuda05/nsg-kitti-car-depth-recon/2023-05-30_182941/new_config.yml --output-path outputs/NSGKITTI-0006-05-260cuda05/nsg-kitti-car-depth-recon/2023-05-30_182941/27w_fulltest.json

my new config just modify some directories to dataset and checkpoint compared to origin config.yml, nothing else
### full eval result
{
  "experiment_name": "0006",
  "method_name": "nsg-kitti-car-depth-recon",
  "checkpoint": "outputs/0006/nsg-kitti-car-depth-recon/2023-08-25_154218/nerfstudio_models/step-000270000.ckpt",
  "results": {
    "psnr": 24.1269474029541,
    "ssim": 0.7862021327018738,
    "lpips": 0.15984047949314117,
    "num_rays_per_sec": 89028.7734375,
    "fps": 0.19115141034126282
  }
}

**Comment by wuzirui** [comment]
Hi, this is a known issue of the checkpoint. We apologize for that. We will upload another one in the next version. Stay tuned :)
